etc/init.d/contextBroker.centos

```diff
@@ -91,7 +91,9 @@ broker_start()
 {
     printf "%-50s" "Starting ${NAME}..."
     if [[ -x ${EXECUTABLE} ]]; then
-        su $BROKER_USER -c "${EXECUTABLE} ${BROKER_OPTS} &> /dev/null "
+        # preserving existing error log (if it already exists)
+        su $BROKER_USER -C "mv ${BROKER_LOG_DIR}.${EXECUTABLE}.err ${BROKER_LOG_DIR}.${EXECUTABLE}.err.old 2> /dev/null || True"
```

Comment:
Also, I wonder if

    || true

should be outside the string to su, like this:

    su $BROKER_USER -c "mv ${BROKER_LOG_DIR}.${EXECUTABLE}.err ${BROKER_LOG_DIR}.${EXECUTABLE}.err.old 2> /dev/null" || true

Not an expert, but ... it doesn't make much sense to me otherwise.
Look it up!